scout/server/blueprints/cases/templates/cases/cases.html

```diff
@@ -115,6 +115,9 @@ <h5 class="modal-title" id="sangerModalLabel">Sanger validations to evaluate:</h
         </div>
       </div>
     </div>
+    <div class="bottom-align-text">
+      <br>Example: case_id=<strong>18201</strong>, HPO-term=<strong>HP:0001166</strong>, synopsis=<strong>synopsis:epilepsy</strong>
```

Comment:
< br>Example: search for a case_id: <strong>18201</strong>, HPO-term: <strong>HP:0001166</strong> or word in synopsis: <strong>synopsis:epilepsy</strong>